sharder/sharded-bot.js

```diff
@@ -96,4 +98,4 @@ const manager = new ShardingManager(startup, {
 
 manager.on('shardCreate', (shard) => console.log(`Shard ${shard.id} launched`));
 
-manager.spawn();
+manager.spawn(totalShards, 5500, -1); 
```

Comment:
I don't know if not having a limit is a great idea. Perhaps just increase it to 1/2 minutes instead?

Reply:
Why wouldn't it be a good idea? 
If the user does not see shards starting they can just stop the process and retry, it's much better than getting errors or having the shards not start because it takes a bit too long, but I mean at the end of the day its on them, we could also just write this down on the docs so people can put whatever time they feel like by themselves

Reply:
I would rather you add a cli option just like shard_count for the timeout, and default it to 30000

Reply:
if I knew how to do that I would, still I believe having it as infinite by default and then letting them change it to a different one  would be better,